src/context/eventQueue.tsx

```diff
@@ -31,14 +33,24 @@ interface EventCallback {
 const eventQueue = new Map<string, EventCallback>();
 
 export const EventQueueProvider: FC<Props> = ({ children }) => {
+  const queryClient = useQueryClient();
+
   return (
     <EventQueueContext.Provider
       value={{
         get: (operationId) => eventQueue.get(operationId),
         set: (operationId, onSuccess, onFailure, onFinish) => {
           eventQueue.set(operationId, { onSuccess, onFailure, onFinish });
+          void queryClient.invalidateQueries({
```

Comment:
good catch! I see this was previously affecting instance creation operations from file uploads